rasa/core/policies/ted_policy.py

```diff
@@ -664,9 +665,15 @@ def run_training(
             self.config[EVAL_NUM_EXAMPLES],
             self.config[RANDOM_SEED],
         )
+
+        current_time = time.strftime("%Y%m%d-%H%M%S")
+        class_name = self.__class__.__name__
+        tensorboard_log_dir = (
+            f"{self.component_config[TENSORBOARD_LOG_DIR]}/{class_name}/{current_time}/"
+        )
```

Comment:
Since the same logic is used across multiple components, lets pull out this logic into a separate function inside `rasa/utils/train_utils.py` which would accept the path to log directory  specified inside the component and the name of the class invoking that function and return the complete path to be used, i.e. `self.component_config[TENSORBOARD_LOG_DIR]}/{class_name}/{current_time}`.

This function can then be invoked by respective components inside their `train` functions to get the correct path that should be passed to `create_common_callback` function.

Also, please use `os.path.join` to construct the complete path. 🙏